pom.xml

```diff
@@ -63,6 +63,8 @@
     <jdk.min.version>11</jdk.min.version>
     <maven.compiler.release>${jdk.min.version}</maven.compiler.release>
     <guava.version>32.1.3-jre</guava.version>
+    <!-- See: https://github.com/SonarSource/sonar-plugin-api#compatibility -->
```

Comment:
I took the version from the documentation, we will run only on the latest 9.9 version update of SQ.